components/home/header/Header.tsx

```diff
@@ -1,18 +1,20 @@
+import Link from "next/link";
 import { Button, Col, Container, Row } from "react-bootstrap";
 import { HeaderImage } from "./HeaderImage";
 import styles from "./Header.module.scss";
-import Link from "next/link";
 
 interface HeaderProps {
-  tagline: string;
+  pluginName: string;
 }
 
-export const Header = ({ tagline }: HeaderProps) => (
+export const Header = ({ pluginName }: HeaderProps) => (
   <Container fluid className={`pipes ${styles.container}`} data-testid="header">
     <Container>
       <Row>
-        <Col md="6" className={`${styles.text}`}>
-          <h2>{tagline}</h2>
+        <Col lg="6" className={`${styles.text}`}>
+          <h2 suppressHydrationWarning>
```

Comment:
This fixes these warnings we've been getting due to the plugin name being randomized

![image](https://user-images.githubusercontent.com/4368928/154349395-dd28fc56-f709-4a84-920b-b6b554930083.png)